openIMIS/openIMIS/settings.py

```diff
@@ -297,6 +299,24 @@ def SITE_URL():
     ],
 }
 
+if MODE == "DEV":
+    # Enhance security in production
+    GRAPHQL_JWT.update({
+        "JWT_COOKIE_SECURE": True,
+        "JWT_COOKIE_SAMESITE": "Lax",
+    })
+
+    CSRF_COOKIE_SECURE = True
+    CSRF_COOKIE_HTTPONLY = True
+    CSRF_COOKIE_SAMESITE = 'Lax'
+
+    SECURE_BROWSER_XSS_FILTER = True
+    SECURE_CONTENT_TYPE_NOSNIFF = True
+    SECURE_HSTS_SECONDS = 63072000
+    SECURE_HSTS_INCLUDE_SUBDOMAINS = True
+    SECURE_HSTS_PRELOAD = True
+    SECURE_SSL_REDIRECT = True
+
```

Comment:
In this piece of code the comment says `# Enhance security in production`. However, this parameters are set only on DEV mode. Can you make adjustments to have consistency?

Reply:
You are right it should be only for PROD